packages/components/src/styles/list/list.declaration.ts

```diff
@@ -0,0 +1,15 @@
+import type { Style } from '../../declaration';
+
+export default {
+  styleName: 'sd-list',
+  summary: 'Generates basic styles for list elements.',
+  status: 'experimental',
+  since: '1.33.1',
+  attributes: [
```

Comment:
Why didn't you create the attributes based on the classes for the sd-list? In this case, please refer to the sd-headline (Docs) for guidance: https://solid-design-system.fe.union-investment.de/x.x.x/storybook/index.html?path=/docs/styles-sd-headline--docs

Reply:
I wasn't aware that the headline is similar to the list and  I don't know exactly what you mean, can we discuss it after the Daily?

Reply:
I added the `attribute` as you suggested.
It's only possible to add the attribute `inverted`, because for the orderded-list or the icon-list there is a diffrent DOM structure and not only another class applied.